spec/system/homepage/user_visits_homepage_spec.rb

```diff
@@ -9,9 +9,9 @@
     it "shows the sign-in block" do
       visit "/"
       within "#sidebar-wrapper-left" do
-        p I18n.t("create_account")
-        expect(page).to have_text(I18n.t("core.log_in"))
-        expect(page).to have_text(I18n.t("core.create_account"))
+        p I18n.t("views.main.header.create_account")
+        expect(page).to have_text(I18n.t("views.main.header.login"))
```

Comment:
confirmed these `views.main.header` keys exist, and match the intent, the original core keys are removed above.